packages/commerce-sdk-react/src/hooks/ShopperProducts/query.ts

```diff
@@ -32,9 +33,6 @@ function useProducts(
     arg: UseProductsArg,
     options?: UseQueryOptions<DataType<Client['getProducts']> | Response, Error>
 ) {
-    if (!arg.ids) {
-        throw new Error('ids is required for useProducts')
```

Comment:
Removing this guard and our own error, so that we're reusing the error that's thrown from the isomorphic sdk.